src/main/java/org/myrobotlab/service/abstracts/AbstractServo.java

```diff
@@ -891,11 +901,11 @@ protected void processMove(Double newPos, boolean blocking, Long timeoutMs) {
       blockingTimeMs = timeEncoder.calculateTrajectory(getPos(), getTargetPos(), getSpeed());
     }
 
-    for (String controller : this.getControllers()) {
-      // TODO: just have a direct reference to the controllers 
-      // avoid this type cast and runtime lookup!!!!!
-      ((ServoController)Runtime.getService(controller)).onServoMoveTo(this);
-    }
+    // GroG: I think in the long run this direct call vs using invoke/send is less preferrable
+    // thinking on a distributed network level you can't do this when the other thing is in
+    // a different process
+    ((ServoController)Runtime.getService(controller)).onServoMoveTo(this);
```

Comment:
for a future pull request.. thanks for adding the comment in the code so we don't forget about it.